flow360/component/flow360_params/flow360_output.py

```diff
@@ -530,10 +532,19 @@ class AeroacousticOutput(Flow360BaseModel, AnimatedOutput):
     """
 
     patch_type: Optional[str] = pd.Field("solid", const=True, alias="patchType")
-    observers: List[Coordinate] = pd.Field()
+    animation_frequency: Optional[PositiveInt] = pd.Field(
```

Comment:
Should this be animated @maciej-flexcompute? Documentation doesn't mention anything: https://docs.flexcompute.com/projects/flow360/en/latest/solverConfiguration/solverConfiguration.html#aeroacousticoutput

Reply:
My bad. I will fix it to be consistent with documentation.

Reply:
Done